### ISSUE TITLE
Playlists has no icons and only black squares when I load them

### ISSUE THREAD

**Issue by Jacalz** [marcrobledo/retroarch-playlist-editor#3]
The playlists just have black squares and no icons when load them in RetroArch on my Nvidia Shield TV. The auto generated playlists made from scanning the directories have the right monocrome icons but these just get black boxes. It is not very pleasant and quite annoying. Is this something I can fix on my own or is this a problem with this great editor?

**Comment by Jacalz** [comment]
Will send screenshots shortly...

**Comment by Jacalz** [comment]
Auto generated playlists from scan directory feature:
![F264_F7_D3-1716-426_C-9_F2_E-_F90_CC3_AF05_E9.jpg](https://s33.postimg.cc/72q14u3rz/F264_F7_D3-1716-426_C-9_F2_E-_F90_CC3_AF05_E9.jpg)

Custom made playlists:
![4_E544_C02-11_F7-4_E19-9_E61-80_B9_C90_FB830.jpg](https://s33.postimg.cc/kjmznwgq7/4_E544_C02-11_F7-4_E19-9_E61-80_B9_C90_FB830.jpg)

**Comment by marcrobledo** [comment]
In order to show the correct icons (both playlist and content), your playlist .lpl file name must match the ones in assets/ directory in RetroArch.
For example, your NES playlist must be named Nintendo - Nintendo Entertainment System.lpl, so it will show the desired icons.
You can use your own custom icons if you place them in assets/ directory and naming it according to your playlists file names.

**Comment by Jacalz** [comment]
Thanks, that solved it :+1:

**Comment by Jacalz** [comment]
Could this be fixed on the website somehow? So you can chose the correct name for the playlist from a drop down menu? It would make it really simple and easy to get icons for new users. :slightly_smiling_face: Also telling the user that he or she need separate playlists per console in order to have the roms separated per console, might be a good idea :wink:

**Comment by marcrobledo** [comment]
Yeah, I would need to think on the best way and the most transparent way for the user to do it.
It could guess the playlist filename with the chosen cores.

**Comment by jeffmadriaga** [comment]
Why do the icons have a dependency on the naming of the playlist. For instance if i want to create a bespoke favourites, or category e.g arcade - racing. Then i am tied to the naming of the core to get the icons display in the UI. Whats the intended design/feature for custom playlists icons?
